frontend/src/components/pages/RegistrantExperience/RegistrationSteps/index.tsx

```diff
@@ -76,8 +77,17 @@ const RegistrationSteps = ({
       optionalClauses: [],
     },
   ]);
+  const [
+    requireEarlyDropOffLatePickup,
+    setRequireEarlyDropOffLatePickup,
+  ] = useState<boolean | null>(null);
+
   const isPersonalInfoFilled = checkPersonalInfoFilled(campers, camp);
-  const isAdditionalInfoFilled = sampleAdditionalInfo;
+  const isAdditionalInfoFilled = checkAdditionalQuestionsAnswered(
+    campers,
+    camp.formQuestions,
```

Comment:
Do we need to filter out the emergency contact questions here? I am currently clarifying which question categories map to which page with Andy so will update you asap afterwards